addon/instance-initializers/clear-double-boot.js

```diff
@@ -0,0 +1,28 @@
+// When using `ember serve` when fastboot addon is installed the application 
+// output will already be rendered to the DOM when the actual JavaScript 
+// loads. Ember does not automatically clear its `rootElement` so this 
+// leads to the "double" applications being visible at once (only the 
+// "bottom" one is running via JS and is interactive).
+//
+// This removes any pre-rendered ember-view elements, so that the booting
+// application will replace the pre-rendered output
+
+export default {
+  name: "clear-double-boot",
+
+  initialize: function(instance) {
+    if (typeof FastBoot === 'undefined') {
+      var originalDidCreateRootView = instance.didCreateRootView;
+
+      instance.didCreateRootView = function() {
+        let elements = document.querySelectorAll(instance.rootElement + ' .ember-view');
```

Comment:
When can we have multiple ember-views?

Reply:
I just moved the code around. I also don't see how this is possible either, but I wouldn't worry because I'm about to change how we mark the beginning and end of the app so fastboot works when the experiemental flag `ember-glimmer-remove-application-template-wrapper` is enabled.
We'll probably need some script tag markets to delimitate the boundaries.